scripts/teardown

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+root="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+
+shopt -s expand_aliases
+alias kp="kubectl --context kind-platform"
+
+main() {
+    if ! kind get clusters | grep -q "platform"; then
+        echo "No kind-platform cluster found; nothing to teardown."
+        exit 0
+    fi
+
+    resources=(promises destinations gitstatestores bucketstatestores)
+    for resource in "${resources[@]}"; do
+        kp delete "$resource" --all
+    done
+
+    kp -n kratix-platform-system delete deployment kratix-platform-controller-manager || true
```

Comment:
deleting a destination doesn't cleanup the bucket for the destination, worth adding some cleanup to the statestore?

Reply:
🤔 i remember adding a line here that did exactly that, but apparently i didn't

will do